Sources/AsyncCollections/TaskQueue.swift

```diff
@@ -15,7 +15,7 @@ import Collections
 /// }
 /// ```
 @available(*, deprecated, message: "Use concurrentMap(maxConcurrentTasks:priority:transform:)")
-public actor TaskQueue<Result> {
+public actor TaskQueue<Result: Sendable> {
```

Comment:
`strict-concurrency=complete` was complaining about `TaskFunc`'s returned type not being Sendable